scripts/core/memory.yml

```diff
@@ -396,7 +398,8 @@ members:
       name: fd
       desc: "[in] the file descriptor handle to import"
 details:
-    - "This structure may be passed to $xMemAllocDevice, via the `pNext` member of $x_device_mem_alloc_desc_t, to import memory from a file descriptor."
+    - "1.0": "This structure may be passed to $xMemAllocDevice, via the `pNext` member of $x_device_mem_alloc_desc_t, to import memory from a file descriptor."
+      "1.5": "This structure may be passed to $xMemAllocDevice or $xMemAllocHost, via the `pNext` member of $x_device_mem_alloc_desc_t or of $x_host_mem_alloc_desc_t, respectively, to import memory from a file descriptor."
```

Comment:
This looks good but I think we need a similar change to `$x_external_memory_import_win32_handle_t` below?

Reply:
@bashbaug missed it. Added.